scilpy/samplingscheme/save_scheme.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import division
+
+import logging
+import numpy as np
+
+# TODO: add some filename extension checking
+
+
+def save_scheme_caru(points, shell_idx, filename, verbose=1):
+    fullfilename = filename + '.caru'
+    f = open(fullfilename, 'w')
+    f.write('# Caruyer format sampling scheme\n')
+    f.write('# X Y Z shell_idx\n')
+    for idx in range(points.shape[0]):
+        f.write('{:.8f} {:.8f} {:.8f} {:.0f}\n'.format(points[idx, 0],
+                                                       points[idx, 1],
+                                                       points[idx, 2],
+                                                       shell_idx[idx]))
+    f.close()
+
+    logging.info('Scheme saved in Caruyer format as {}'.format(fullfilename))
+
+
+def save_scheme_philips(points, shell_idx, bvalues, filename, verbose=1):
+    fullfilename = filename + '.txt'
+    f = open(fullfilename, 'w')
+    f.write('# Philips format sampling scheme\n')
+    f.write('# X Y Z bval\n')
+    for idx in range(points.shape[0]):
+        f.write('{:.3f} {:.3f} {:.3f} {:.2f}\n'.format(points[idx, 0],
+                                                       points[idx, 1],
+                                                       points[idx, 2],
+                                                       bvalues[shell_idx[idx]]))
+    f.close()
+
+    logging.info('Scheme saved in Philips format as {}'.format(fullfilename))
+
+
+def save_scheme_mrtrix(points, shell_idx, bvalues, filename, verbose=1):
+    fullfilename = filename + '.b'
+    f = open(fullfilename, 'w')
+    for idx in range(points.shape[0]):
+        f.write('{:.8f},{:.8f},{:.8f},{:.8f}\n'.format(points[idx, 0],
```

Comment:
I know this is not new, but this doesn't respect the Mrtrix standard. Additionally, if we save a sampling scheme with this, and then try to convert to FSL, it will crash because the parsing is not ok.